routers/web/user/home.go

```diff
@@ -822,6 +822,11 @@ func UsernameSubRoute(ctx *context.Context) {
 	reloadParam := func(suffix string) (success bool) {
 		ctx.SetParams("username", strings.TrimSuffix(username, suffix))
 		context_service.UserAssignmentWeb()(ctx)
+		// check view permissions
+		if ctx.ContextUser.IsIndividual() && !user_model.IsUserVisibleToViewer(ctx, ctx.ContextUser, ctx.Doer) {
```

Comment:
```suggestion
		if !user_model.IsUserVisibleToViewer(ctx, ctx.ContextUser, ctx.Doer) {
```

Reply:
does the IsUserVisibleToViewer handle orgs correctly?

Reply:
1. ~~I hope so~~ Edit: Yes, I checked, it does, it's simply a little inefficient.
2. I would argue that your check is rather erroneous.